src/status_im/common/check_before_syncing/view.cljs

```diff
@@ -0,0 +1,70 @@
+(ns status-im.common.check-before-syncing.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.common.check-before-syncing.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def checks
+  [(i18n/label :t/check-before-syncing-doc-checkbox-1)
+   (i18n/label :t/check-before-syncing-doc-checkbox-2)
+   (i18n/label :t/check-before-syncing-doc-checkbox-3)])
+
+(defn- checkbox
+  [on-change description]
+  [rn/view {:style style/checkbox}
+   [quo/selectors
+    {:type      :checkbox
+     :blur?     true
+     :on-change on-change}]
+   [quo/text {:style style/checkbox-text} description]])
+
+(defn- compute-new-checked-count
+  [set-checked-count checked?]
+  (if checked?
+    (set-checked-count #(inc %))
+    (set-checked-count #(dec %))))
```

Comment:
could just use `(set-checked-count inc)` and same for `dec`?

Reply:
fixed, I thought there's another arg after prevState and turns out there's not